code/build/android/jni/Application.mk

```diff
@@ -1,5 +1,5 @@
-APP_PLATFORM := android-14
+APP_PLATFORM := android-19
```

Comment:
Hi @danvd. Is it possible to keep Ice cream sandwich support back to 14? Or does it definitely have to move forward to 19. All good if so. Thanks!

Reply:
since, there is no code change, API14 should still works fine
that being said, API19 is Android 4.4 and, nowadays, supporting below 4.4 doesn't make a big difference

Reply:
I got `Android NDK: android-14 is unsupported. Using minimum supported version android-16.` when building with the latest NDK. So I guess it now has to be minimum 16.

Reply:
As per @danvd, `abs` is missing from `libc` until API 19.